tools/interactive/interactivetool_cellprofiler.xml

```diff
@@ -1,5 +1,5 @@
-<tool id="interactive_tool_cellprofiler" tool_type="interactive" name="cellprofiler" version="@VERSION@+galaxy0" profile="23.1">
-	<description>cell image analysis software</description>
+<tool id="interactive_tool_cellprofiler" tool_type="interactive" name="Run CellProfiler" version="@VERSION@+galaxy0" profile="23.1">
+	<description>interactive tool</description>
```

Comment:
interactive tool is a very technical term ... I liked the previous version more. Happy to merge as it is if you prefer this version.

Reply:
We discussed it here with Leonid. To follow the [naming convention agreed at the BH](https://github.com/elixir-europe/biohackathon-projects-2023/blob/main/16/conventions.md) and distinguish it from the the CellProfiler tools, that seems to be the best option. 

Cause it does not look very self-explanatory now:
<img width="287" alt="Screenshot 2024-02-29 at 15 53 40" src="https://github.com/usegalaxy-eu/galaxy/assets/11300231/28fb66fe-e5f2-4e35-8701-e6708040f8ca">